python/tests/tests.py

```diff
@@ -13,7 +13,10 @@
 # limitations under the License.
 #
 
+import shutil
```

Comment:
Note: I moved logic for creating/cleaning up a temp directory for storing exported Keras models into the base test class (`SparkDLTestCase`), should I maybe create a new class for this?

Reply:
yeah probably better to create a new class. also i think it's okay to put it in the same file.